### PR TITLE
Config service - Solution settings tests

### PR DESCRIPTION
**Pull request by isaac73**
# Description and Motivation 
Config service - Solution settings tests
...

# Change type 

- [ ] Bug fix
- [x] New feature
- [ ] Enhancement
- [ ] Breaking change (breaks backward compatibility)

**Checklist:**

- [x] All tests passed
- [x] The code follows the code style and conventions of this project
- [ ] The change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: Config/SolutionSettingsTests.cs [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+        /// <summary>
+        /// Test that the service can set a solution-settings file in CosmosDB
+        /// </summary>
+        [Fact, Trait(Constants.TEST, Constants.INTEGRATION_TEST)]
+        public void ShouldSetSolutionSettingsLogo()
+        {
+            var request = new HttpRequest();
+            request.Options.Timeout = TIMEOUT_MS;
+            request.SetUriFromString(Constants.CONFIG_ADDRESS + COLLECTION_ID + LOGO_KEY);
+            request.SetContent(TEST_JSON);
```
Comment (molly-moen) [inline Config/SolutionSettingsTests.cs]: logo isn't uploaded as json, it's uploaded as an image binary

File: ASAManager/run.sh [hunk 2]
```diff
@@ -16,9 +16,9 @@ start_containers() {
     cd $APP_HOME
     ./scripts/storageadapter.sh start
     ./scripts/telemetry.sh start
+    ./scripts/devicesimulation.sh start
```
Comment (molly-moen) [inline ASAManager/run.sh]: why this change?

Comment (isaac73) [reply ASAManager/run.sh]: pcs config depends on storage adapter, telemetry and device simulation and hence this change

File: Config/SolutionSettingsTests.cs (hunk 1 above)
Comment (isaac73) [reply Config/SolutionSettingsTests.cs]: done
